app/src/main/java/org/fossasia/openevent/app/data/network/api/RetrofitEventRepository.java

```diff
@@ -0,0 +1,87 @@
+package org.fossasia.openevent.app.data.network.api;
+
+import org.fossasia.openevent.app.contract.model.ICacheModel;
+import org.fossasia.openevent.app.contract.model.IEventRepository;
+import org.fossasia.openevent.app.contract.model.IUtilModel;
+import org.fossasia.openevent.app.data.cache.ObjectCache;
+import org.fossasia.openevent.app.data.models.Attendee;
+import org.fossasia.openevent.app.data.models.Event;
+import org.fossasia.openevent.app.data.models.User;
+import org.fossasia.openevent.app.utils.Constants;
+
+import java.util.List;
+
+import io.reactivex.Observable;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+
+public class RetrofitEventRepository implements IEventRepository {
```

Comment:
@iamareebjamal same here. `EventDataRepository` would be enough. Just name the related interface as `EventDataSource`.